core/src/ledger/storage/local_node.rs

```diff
@@ -0,0 +1,114 @@
+//! Functionality to do with persisting data related to the local node
+
+use borsh::{BorshDeserialize, BorshSerialize};
+
+use super::{DBIter, Storage, StorageHasher, DB};
+use crate::ledger::storage_api::{StorageRead, StorageWrite};
+use crate::types::ethereum;
+use crate::types::storage::{self, DbKeySeg};
+
+/// Represents a value in storage which is to do with our local node, rather
+/// than any specific chain.
+pub trait LocalNodeValue<T: BorshSerialize + BorshDeserialize> {
+    /// What the initial value for this key should be when a chain is created.
+    fn initial_value(&self) -> T;
+    /// The storage key under which this value is stored.
+    fn key(&self) -> storage::Key;
+}
+
+/// Represents the last Ethereum block height that the Ethereum oracle fully
+/// processed.
+#[derive(Debug)]
+pub struct EthereumOracleLastProcessedBlock;
```

Comment:
There are other local node values we currently store in RocksDB (e.g. `tx_queue`)